File: Practice/21/Python/Project.py
def bmi(weight: float, height: float) -> float:
    return weight/(height/100)**2

def print_bmi(bmi: float) -> float:
    if bmi < 18.5:
        print("Underweight")
    elif bmi >= 18.5 and bmi < 25.0:
        print("Normal")
    elif bmi >= 25.0 and bmi < 30.0:
        print("Overweight")
    elif bmi >= 30.0:
        print("Obesity")
        


w = float(input('Enter your weight: '))
h = float(input('Enter your height: '))
print_bmi(bmi(w, h)) 

File: Practice/21/C++/Project/Project/Project.cpp
#include <iostream>


double BMI(double weight, double height);
void printBMI(double BMI);

int main()
{
	double w, h;
	std::cout << "Enter your weight: ";
	std::cin >> w;
	std::cout << "Enter your height: ";
	std::cin >> h;

	printBMI(BMI(w, h));
}
double BMI(double weight, double height) {
	return (weight / pow((height/100), 2));
}
void printBMI(double BMI) {
	if (BMI < 18.5) std::cout << "Underweight\n";
	else if (BMI >= 18.5 && BMI < 25.0) std::cout << "Normal\n";
	else if (BMI >= 25.0 && BMI < 30.0) std::cout << "Overweight\n";
	else if (BMI >= 30.0) std::cout << "Obesity\n";
	return;
}
